api/v1beta2/ocirepository_types.go

```diff
@@ -148,10 +152,12 @@ type OCILayerSelector struct {
 type OCIRepositoryVerification struct {
 	// Provider specifies the technology used to sign the OCI Artifact.
 	// +kubebuilder:validation:Enum=cosign
+	// +kubebuilder:default:=cosign
 	Provider string `json:"provider"`
 
 	// SecretRef specifies the Kubernetes Secret containing the
 	// trusted public keys.
+	// +required
```

Comment:
We don’t want to make the secret required, when not specified we can try keyless.

Reply:
removed, and keyless verification logic added, thx.

Reply:
PTAL @Dentrax